Caboodle/Types/Placemark.shared.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Caboodle
+{
+    public class Placemark
```

Comment:
Could we maybe use the naming from UWP?
https://docs.microsoft.com/en-us/uwp/api/windows.services.maps.mapaddress

I find these properties very hard to understand. If I want the street number, I have to use the weird `SubThoroughfare` (which sounds like a highway or something) - UWP calls it `StreetNumber`.

Reply:
Anyone have opinions on this?